README.md

```diff
@@ -18,3 +18,11 @@ This repository contains multiple packages:
 - [`@itwin/imodels-client-common-config`](utils/imodels-client-common-config/README.md) package is used internally to share common configuration across the API clients.
 - [`@itwin/imodels-clients-tests`](tests/imodels-clients-tests/README.md) package is used internally for `@itwin/imodels-client-management` and `@itwin/imodels-client-authoring` package testing.
 - [`@itwin/imodels-access-backend-tests`](tests/imodels-access-backend-tests/README.md) package is used internally for API client testing.
+
+## iTwin.js versions supported by @itwin/imodels-access-* packages
+| iTwin.js version | @itwin/imodels-access-* version |
+|------------------|---------------------------------|
+| ^3.0.0           | >=1.0.1 <2.1.0                  |
```

Comment:
2.0.0, not 2.1.0. it is confusing looking at the history but we only released 2.0.0 after we upgraded to itwin.js 3.3.0, you can double check that by downloading the specific package version and opening the `package.json`
`npm i @itwin/imodels-access-backend@2.0.0`

maybe it would also be good to mention 5.0.0 version of clients in the last line
```
| ^4.0.0 | >=4.0.0, >=5.0.0  |
```

Reply:
@aruniverse fyi

Reply:
thanks, updated